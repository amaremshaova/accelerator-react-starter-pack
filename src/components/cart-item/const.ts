import { ProductsInCart } from '../../types/cart';

export const getTotal = (guitarsInCart : ProductsInCart[]) => {
  let total = 0;
  guitarsInCart.forEach((item) => total += item.product.price * item.count);

  return total;
};
