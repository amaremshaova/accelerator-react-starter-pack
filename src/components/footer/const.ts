export enum SocialName {
  Instagram = 'instagram',
  Facebook = 'facebook',
  Twitter = 'twitter'
}

export enum SocialLink {
  Instagram = 'https://www.instagram.com/',
  Facebook = 'https://www.facebook.com/',
  Twitter = 'https://www.twitter.com/'
}

export enum Contact {
  City = 'Санкт-Петербург',
  Phone = '8-812-500-50-50',
  Metro = 'м. Невский проспект',
  Address = 'ул. Казанская 6',
  OpeningHours = 'с 11:00 до 20:00',
  NumberWorkingDays = 'без выходных'
}
