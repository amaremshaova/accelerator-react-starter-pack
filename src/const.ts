export enum ApiPath{
  Guitars = '/guitars/',
  Comments = '/comments',
  Coupons = '/coupons',
  Orders = '/orders',
}


export enum AppRoute {
  Main = '#',
  Catalog = '/catalog/',
  CatalogPage = '/catalog/page_',
  Empty = '#',
  CatalogStartPage = '/catalog/page_1',
  Cart = '/cart',
  Undefined = '/*',
  Root = '/',
}

