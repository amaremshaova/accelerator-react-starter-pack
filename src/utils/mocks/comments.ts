import { GuitarType } from '../../components/filters/filters';
import { GuitarComments } from '../../types/guitar-comments';

export const commentsCount = 10;
export const commentsCountArray = [{id: 1, count: 10}, {id: 2, count:20}];

export const comments = [{
  id: '1',
  userName: 'a',
  advantage: 'хороший звук',
  disadvantage: 'не очень качественный корпус',
  comment: 'В целом, понравилось',
  rating: 3.5,
  createAt: '',
  guitarId: 100,
},
{
  id: '2',
  userName: 'b',
  advantage: 'нормальный звук',
  disadvantage: 'не обнаружено',
  comment: 'Подходит',
  rating: 5,
  createAt: '',
  guitarId: 100,
}];

export const guitarComments : GuitarComments[] = [
  {
    id: 1,
    name: 'Честер Bass',
    vendorCode: 'SO757575',
    type: GuitarType.Electric,
    description: 'Замечательный малобюджетный вариант, созданный для новичков, которые отдают предпочтение мелодичным стилям. Прекрасно звучат блюз и баллады, исполненные на этой гитаре. Акустические свойства весьма высоки, в отличие от ее стоимости.',
    previewImg: 'img/guitar-1.jpg',
    stringCount: 7,
    rating: 3.5,
    price: 17500,
    comments: [{
      id: '1',
      userName: 'a',
      advantage: 'хороший звук',
      disadvantage: 'не очень качественный корпус',
      comment: 'В целом, понравилось',
      rating: 3.5,
      createAt: '',
      guitarId: 100,
    }],
  },
];
