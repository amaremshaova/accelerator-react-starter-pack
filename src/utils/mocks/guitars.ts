import { GuitarType } from '../../components/filters/filters';
import { Guitar } from '../../types/guitar';

export const guitars : Guitar[] = [
  {
    id: 1,
    name: 'Честер Bass',
    vendorCode: 'SO757575',
    type: GuitarType.Electric,
    description: 'Замечательный малобюджетный вариант, созданный для новичков, которые отдают предпочтение мелодичным стилям. Прекрасно звучат блюз и баллады, исполненные на этой гитаре. Акустические свойства весьма высоки, в отличие от ее стоимости.',
    previewImg: 'img/guitar-1.jpg',
    stringCount: 7,
    rating: 3.5,
    price: 17500,
  },
  {
    id: 2,
    name: 'Честер Z300',
    vendorCode: 'TK129049',
    type: GuitarType.Electric,
    description: 'Эргономичность гитары и качество сборки являются, пожалуй, её главными преимуществами. Идеальное расположение в руках музыканта дополняется прочностью конструкции из клёна.',
    previewImg: 'img/guitar-8.jpg',
    stringCount: 7,
    rating: 3.5,
    price: 29500,
  }];


export const guitarsCount = 2;

export const minPrice  = 17500;
export const maxPrice  = 29500;
