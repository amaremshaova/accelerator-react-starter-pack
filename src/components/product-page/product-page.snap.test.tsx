import {render} from '@testing-library/react';
import {Route, Router, Routes} from 'react-router-dom';
import { Provider } from 'react-redux';
import { guitars, maxPrice, minPrice } from '../../utils/mocks/guitars';
import { commentsCountArray } from '../../utils/mocks/comments';
import { configureMockStore } from '@jedmao/redux-mock-store';
import { createMemoryHistory } from 'history';
import { AppRoute } from '../../const';
import thunk from 'redux-thunk';
import ProductPage from './product-page';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

const store = mockStore({
  DATA: {guitars: guitars,
    likeGuitars: guitars,
    guitarsCount: 27,
    minPrice: minPrice,
    maxPrice: maxPrice,
    commentsCount: commentsCountArray},
  APP : {activeProduct: guitars[0], productsInCart: []},
});

const history = createMemoryHistory();


describe('Component: ProductPage', () => {
  it('should render correctly', () => {

    history.push(AppRoute.Catalog + 1);

    window.scroll = jest.fn(()=>'w');

    const {container} = render(
      <Provider store = {store}>
        <Router navigator={history} location={history.location}>
          <Routes>
            <Route path={AppRoute.Catalog + 1} element={ <ProductPage/>}/>
          </Routes>
        </Router>
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
