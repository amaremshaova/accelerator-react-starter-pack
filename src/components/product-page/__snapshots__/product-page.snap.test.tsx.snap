// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ProductPage should render correctly 1`] = `
<div>
  <header
    class="header"
    id="header"
  >
    <div
      class="container header__wrapper"
    >
      <a
        class="header__logo logo"
        href="/catalog/1"
      >
        <img
          alt="Логотип"
          class="logo__img"
          height="70"
          src="./img/svg/logo.svg"
          width="70"
        />
      </a>
      <nav
        class="main-nav"
      >
        <ul
          class="main-nav__list"
        >
          <li>
            <a
              class="link main-nav__link"
              href="/catalog/page_1"
            >
              Каталог
            </a>
          </li>
          <li>
            <a
              class="link main-nav__link"
              href="/catalog/1"
            >
              Где купить?
            </a>
          </li>
          <li>
            <a
              class="link main-nav__link"
              href="/catalog/1"
            >
              О компании
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="form-search"
      >
        <form
          class="form-search__form"
          data-testid="form-search__form"
        >
          <button
            class="form-search__submit"
            type="submit"
          >
            <svg
              aria-hidden="true"
              class="form-search__icon"
              height="15"
              width="14"
            >
              <use
                href="#icon-search"
              />
            </svg>
            <span
              class="visually-hidden"
            >
              Начать поиск
            </span>
          </button>
          <input
            autocomplete="off"
            class="form-search__input"
            data-testid="search"
            id="search"
            placeholder="что вы ищете?"
            type="text"
            value=""
          />
          <label
            class="visually-hidden"
            for="search"
          >
            Поиск
          </label>
        </form>
        <ul
          class="form-search__select-list
        hidden"
        />
      </div>
      <a
        aria-label="Корзина"
        class="header__cart-link"
        href="/cart"
      >
        <svg
          aria-hidden="true"
          class="header__cart-icon"
          height="14"
          width="14"
        >
          <use
            href="#icon-basket"
          />
        </svg>
        <span
          class="visually-hidden"
        >
          Перейти в корзину
        </span>
        
      </a>
    </div>
  </header>
  <main
    class="page-content"
  >
    <div
      class="container"
    >
      <h1
        class="page-content__title title title--bigger"
      >
        Честер Bass
      </h1>
      <ul
        class="breadcrumbs page-content__breadcrumbs"
      >
        <li
          class="breadcrumbs__item"
        >
          <a
            class="link"
            href="/catalog/1"
            tabindex="0"
          >
            Главная
          </a>
        </li>
        <li
          class="breadcrumbs__item"
        >
          <a
            class="link"
            href="/catalog/page_1"
            tabindex="0"
          >
            Каталог
          </a>
        </li>
        <li
          class="breadcrumbs__item"
        >
          <a
            class="link"
            href="/catalog/1"
            tabindex="0"
          >
            Честер Bass
          </a>
        </li>
      </ul>
      <div
        class="product-container"
      >
        <img
          alt=""
          class="product-container__img"
          height="235"
          src="img/guitar-1.jpg"
          width="90"
        />
        <div
          class="product-container__info-wrapper"
        >
          <h2
            class="product-container__title title title--big title--uppercase"
          >
            Честер Bass
          </h2>
          <div
            aria-hidden="true"
            class="rate product-container__rating"
          >
            <span
              class="visually-hidden"
            >
              Рейтинг:
            </span>
            <svg
              aria-hidden="true"
              data-testid="100-icon"
              height="14"
              width="14"
            >
              <lineargradient
                height="8"
                id="lg0-1"
                width="9"
                x1="0%"
                x2="100%"
                y1="0%"
                y2="0%"
              >
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
              </lineargradient>
              <path
                d="M11.481 4.14845L8.65177 3.75745L7.38722 1.32696C7.16064 0.893731 6.50416 0.888224 6.27565 1.32696L5.01109 3.75745L2.18182 4.14845C1.67445 4.21821 1.47111 4.81115 1.83905 5.15075L3.88597 7.04154L3.40183 9.7125C3.31469 10.1953 3.85111 10.5569 4.30038 10.3311L6.83143 9.07L9.36248 10.3311C9.81176 10.5551 10.3482 10.1953 10.261 9.7125L9.7769 7.04154L11.8238 5.15075C12.1918 4.81115 11.9884 4.21821 11.481 4.14845ZM8.77958 6.73314L9.23854 9.27377L6.83143 8.07505L4.42432 9.27377L4.88328 6.73314L2.93513 4.93414L5.62691 4.56332L6.83143 2.25032L8.03595 4.56332L10.7277 4.93414L8.77958 6.73314Z"
                fill="#C90606"
              />
              <path
                d="M12.0902 4.13787L8.97645 3.70544L7.58451 1.00895C7.54649 0.935124 7.48394 0.875359 7.40668 0.83903C7.21291 0.747624 6.97745 0.823796 6.88056 1.00895L5.48862 3.70544L2.37484 4.13787C2.289 4.14959 2.21051 4.18826 2.15041 4.24685C2.07777 4.3182 2.03773 4.4142 2.03911 4.51374C2.04049 4.61328 2.08317 4.70822 2.15777 4.77771L4.41063 6.87654L3.87838 9.84022C3.8659 9.90917 3.87389 9.98007 3.90143 10.0449C3.92897 10.1097 3.97498 10.1659 4.03422 10.207C4.09346 10.2481 4.16358 10.2725 4.23661 10.2775C4.30964 10.2825 4.38268 10.2678 4.44743 10.2351L7.23254 8.83593L10.0176 10.2351C10.0937 10.2738 10.182 10.2867 10.2666 10.2726C10.48 10.2375 10.6235 10.0441 10.5867 9.84022L10.0544 6.87654L12.3073 4.77771C12.3686 4.72029 12.4091 4.64529 12.4214 4.56326C12.4545 4.35818 12.3048 4.16834 12.0902 4.13787Z"
                fill="url(#lg0-1)"
              />
            </svg>
            <svg
              aria-hidden="true"
              data-testid="100-icon"
              height="14"
              width="14"
            >
              <lineargradient
                height="8"
                id="lg1-1"
                width="9"
                x1="0%"
                x2="100%"
                y1="0%"
                y2="0%"
              >
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
              </lineargradient>
              <path
                d="M11.481 4.14845L8.65177 3.75745L7.38722 1.32696C7.16064 0.893731 6.50416 0.888224 6.27565 1.32696L5.01109 3.75745L2.18182 4.14845C1.67445 4.21821 1.47111 4.81115 1.83905 5.15075L3.88597 7.04154L3.40183 9.7125C3.31469 10.1953 3.85111 10.5569 4.30038 10.3311L6.83143 9.07L9.36248 10.3311C9.81176 10.5551 10.3482 10.1953 10.261 9.7125L9.7769 7.04154L11.8238 5.15075C12.1918 4.81115 11.9884 4.21821 11.481 4.14845ZM8.77958 6.73314L9.23854 9.27377L6.83143 8.07505L4.42432 9.27377L4.88328 6.73314L2.93513 4.93414L5.62691 4.56332L6.83143 2.25032L8.03595 4.56332L10.7277 4.93414L8.77958 6.73314Z"
                fill="#C90606"
              />
              <path
                d="M12.0902 4.13787L8.97645 3.70544L7.58451 1.00895C7.54649 0.935124 7.48394 0.875359 7.40668 0.83903C7.21291 0.747624 6.97745 0.823796 6.88056 1.00895L5.48862 3.70544L2.37484 4.13787C2.289 4.14959 2.21051 4.18826 2.15041 4.24685C2.07777 4.3182 2.03773 4.4142 2.03911 4.51374C2.04049 4.61328 2.08317 4.70822 2.15777 4.77771L4.41063 6.87654L3.87838 9.84022C3.8659 9.90917 3.87389 9.98007 3.90143 10.0449C3.92897 10.1097 3.97498 10.1659 4.03422 10.207C4.09346 10.2481 4.16358 10.2725 4.23661 10.2775C4.30964 10.2825 4.38268 10.2678 4.44743 10.2351L7.23254 8.83593L10.0176 10.2351C10.0937 10.2738 10.182 10.2867 10.2666 10.2726C10.48 10.2375 10.6235 10.0441 10.5867 9.84022L10.0544 6.87654L12.3073 4.77771C12.3686 4.72029 12.4091 4.64529 12.4214 4.56326C12.4545 4.35818 12.3048 4.16834 12.0902 4.13787Z"
                fill="url(#lg1-1)"
              />
            </svg>
            <svg
              aria-hidden="true"
              data-testid="100-icon"
              height="14"
              width="14"
            >
              <lineargradient
                height="8"
                id="lg2-1"
                width="9"
                x1="0%"
                x2="100%"
                y1="0%"
                y2="0%"
              >
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
              </lineargradient>
              <path
                d="M11.481 4.14845L8.65177 3.75745L7.38722 1.32696C7.16064 0.893731 6.50416 0.888224 6.27565 1.32696L5.01109 3.75745L2.18182 4.14845C1.67445 4.21821 1.47111 4.81115 1.83905 5.15075L3.88597 7.04154L3.40183 9.7125C3.31469 10.1953 3.85111 10.5569 4.30038 10.3311L6.83143 9.07L9.36248 10.3311C9.81176 10.5551 10.3482 10.1953 10.261 9.7125L9.7769 7.04154L11.8238 5.15075C12.1918 4.81115 11.9884 4.21821 11.481 4.14845ZM8.77958 6.73314L9.23854 9.27377L6.83143 8.07505L4.42432 9.27377L4.88328 6.73314L2.93513 4.93414L5.62691 4.56332L6.83143 2.25032L8.03595 4.56332L10.7277 4.93414L8.77958 6.73314Z"
                fill="#C90606"
              />
              <path
                d="M12.0902 4.13787L8.97645 3.70544L7.58451 1.00895C7.54649 0.935124 7.48394 0.875359 7.40668 0.83903C7.21291 0.747624 6.97745 0.823796 6.88056 1.00895L5.48862 3.70544L2.37484 4.13787C2.289 4.14959 2.21051 4.18826 2.15041 4.24685C2.07777 4.3182 2.03773 4.4142 2.03911 4.51374C2.04049 4.61328 2.08317 4.70822 2.15777 4.77771L4.41063 6.87654L3.87838 9.84022C3.8659 9.90917 3.87389 9.98007 3.90143 10.0449C3.92897 10.1097 3.97498 10.1659 4.03422 10.207C4.09346 10.2481 4.16358 10.2725 4.23661 10.2775C4.30964 10.2825 4.38268 10.2678 4.44743 10.2351L7.23254 8.83593L10.0176 10.2351C10.0937 10.2738 10.182 10.2867 10.2666 10.2726C10.48 10.2375 10.6235 10.0441 10.5867 9.84022L10.0544 6.87654L12.3073 4.77771C12.3686 4.72029 12.4091 4.64529 12.4214 4.56326C12.4545 4.35818 12.3048 4.16834 12.0902 4.13787Z"
                fill="url(#lg2-1)"
              />
            </svg>
            <svg
              aria-hidden="true"
              data-testid="50-icon"
              height="14"
              width="14"
            >
              <lineargradient
                height="8"
                id="lg3-1"
                width="9"
                x1="0%"
                x2="100%"
                y1="0%"
                y2="0%"
              >
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="50%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="50%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
              </lineargradient>
              <path
                d="M11.481 4.14845L8.65177 3.75745L7.38722 1.32696C7.16064 0.893731 6.50416 0.888224 6.27565 1.32696L5.01109 3.75745L2.18182 4.14845C1.67445 4.21821 1.47111 4.81115 1.83905 5.15075L3.88597 7.04154L3.40183 9.7125C3.31469 10.1953 3.85111 10.5569 4.30038 10.3311L6.83143 9.07L9.36248 10.3311C9.81176 10.5551 10.3482 10.1953 10.261 9.7125L9.7769 7.04154L11.8238 5.15075C12.1918 4.81115 11.9884 4.21821 11.481 4.14845ZM8.77958 6.73314L9.23854 9.27377L6.83143 8.07505L4.42432 9.27377L4.88328 6.73314L2.93513 4.93414L5.62691 4.56332L6.83143 2.25032L8.03595 4.56332L10.7277 4.93414L8.77958 6.73314Z"
                fill="#C90606"
              />
              <path
                d="M12.0902 4.13787L8.97645 3.70544L7.58451 1.00895C7.54649 0.935124 7.48394 0.875359 7.40668 0.83903C7.21291 0.747624 6.97745 0.823796 6.88056 1.00895L5.48862 3.70544L2.37484 4.13787C2.289 4.14959 2.21051 4.18826 2.15041 4.24685C2.07777 4.3182 2.03773 4.4142 2.03911 4.51374C2.04049 4.61328 2.08317 4.70822 2.15777 4.77771L4.41063 6.87654L3.87838 9.84022C3.8659 9.90917 3.87389 9.98007 3.90143 10.0449C3.92897 10.1097 3.97498 10.1659 4.03422 10.207C4.09346 10.2481 4.16358 10.2725 4.23661 10.2775C4.30964 10.2825 4.38268 10.2678 4.44743 10.2351L7.23254 8.83593L10.0176 10.2351C10.0937 10.2738 10.182 10.2867 10.2666 10.2726C10.48 10.2375 10.6235 10.0441 10.5867 9.84022L10.0544 6.87654L12.3073 4.77771C12.3686 4.72029 12.4091 4.64529 12.4214 4.56326C12.4545 4.35818 12.3048 4.16834 12.0902 4.13787Z"
                fill="url(#lg3-1)"
              />
            </svg>
            <svg
              aria-hidden="true"
              data-testid="0-icon"
              height="14"
              width="14"
            >
              <lineargradient
                height="8"
                id="lg4-1"
                width="9"
                x1="0%"
                x2="100%"
                y1="0%"
                y2="0%"
              >
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="1"
                />
                <stop
                  offset="0%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
                <stop
                  offset="100%"
                  stop-color="#C90606"
                  stop-opacity="0"
                />
              </lineargradient>
              <path
                d="M11.481 4.14845L8.65177 3.75745L7.38722 1.32696C7.16064 0.893731 6.50416 0.888224 6.27565 1.32696L5.01109 3.75745L2.18182 4.14845C1.67445 4.21821 1.47111 4.81115 1.83905 5.15075L3.88597 7.04154L3.40183 9.7125C3.31469 10.1953 3.85111 10.5569 4.30038 10.3311L6.83143 9.07L9.36248 10.3311C9.81176 10.5551 10.3482 10.1953 10.261 9.7125L9.7769 7.04154L11.8238 5.15075C12.1918 4.81115 11.9884 4.21821 11.481 4.14845ZM8.77958 6.73314L9.23854 9.27377L6.83143 8.07505L4.42432 9.27377L4.88328 6.73314L2.93513 4.93414L5.62691 4.56332L6.83143 2.25032L8.03595 4.56332L10.7277 4.93414L8.77958 6.73314Z"
                fill="#C90606"
              />
              <path
                d="M12.0902 4.13787L8.97645 3.70544L7.58451 1.00895C7.54649 0.935124 7.48394 0.875359 7.40668 0.83903C7.21291 0.747624 6.97745 0.823796 6.88056 1.00895L5.48862 3.70544L2.37484 4.13787C2.289 4.14959 2.21051 4.18826 2.15041 4.24685C2.07777 4.3182 2.03773 4.4142 2.03911 4.51374C2.04049 4.61328 2.08317 4.70822 2.15777 4.77771L4.41063 6.87654L3.87838 9.84022C3.8659 9.90917 3.87389 9.98007 3.90143 10.0449C3.92897 10.1097 3.97498 10.1659 4.03422 10.207C4.09346 10.2481 4.16358 10.2725 4.23661 10.2775C4.30964 10.2825 4.38268 10.2678 4.44743 10.2351L7.23254 8.83593L10.0176 10.2351C10.0937 10.2738 10.182 10.2867 10.2666 10.2726C10.48 10.2375 10.6235 10.0441 10.5867 9.84022L10.0544 6.87654L12.3073 4.77771C12.3686 4.72029 12.4091 4.64529 12.4214 4.56326C12.4545 4.35818 12.3048 4.16834 12.0902 4.13787Z"
                fill="url(#lg4-1)"
              />
            </svg>
            <span
              class="rate__count"
            >
              10
            </span>
            <span
              class="rate__message"
            />
          </div>
          <div
            class="tabs"
          >
            <a
              class="button button--medium tabs__button
            "
              href="#characteristics"
            >
              Характеристики
            </a>
            <a
              class="button button--medium tabs__button
              button--black-border"
              href="#description"
            >
              Описание
            </a>
            <div
              class="tabs__content"
              id="characteristics"
            >
              <table
                class="tabs__table "
              >
                <thead>
                  <tr
                    class="tabs__table-row"
                  >
                    <td
                      class="tabs__title"
                    >
                      Артикул:
                    </td>
                    <td
                      class="tabs__value"
                    >
                      SO757575
                    </td>
                  </tr>
                  <tr
                    class="tabs__table-row"
                  >
                    <td
                      class="tabs__title"
                    >
                      Тип:
                    </td>
                    <td
                      class="tabs__value"
                    >
                      Электрогитара
                    </td>
                  </tr>
                  <tr
                    class="tabs__table-row"
                  >
                    <td
                      class="tabs__title"
                    >
                      Количество струн:
                    </td>
                    <td
                      class="tabs__value"
                    >
                      7
                       струнная
                    </td>
                  </tr>
                </thead>
              </table>
              <p
                class="tabs__product-description  hidden"
              >
                Замечательный малобюджетный вариант, созданный для новичков, которые отдают предпочтение мелодичным стилям. Прекрасно звучат блюз и баллады, исполненные на этой гитаре. Акустические свойства весьма высоки, в отличие от ее стоимости.
              </p>
            </div>
          </div>
        </div>
        <div
          class="product-container__price-wrapper"
        >
          <p
            class="product-container__price-info product-container__price-info--title"
          >
            Цена:
          </p>
          <p
            class="product-container__price-info product-container__price-info--value"
          >
            17 500
             ₽
          </p>
          <a
            class="button button--red button--big product-container__button"
            href="#"
          >
            Добавить в корзину
          </a>
        </div>
      </div>
      <section
        class="reviews"
      >
        <h3
          class="reviews__title title title--bigger"
        >
          Отзывы
        </h3>
        <a
          class="button button--red-border button--big reviews__sumbit-button"
          href="#"
        >
          Оставить отзыв
        </a>
        <p
          style="margin-top: 200px; font-size: 30px;"
        >
          Загрузка отзывов...
        </p>
        
        
      </section>
    </div>
  </main>
  <footer
    class="footer"
  >
    <div
      class="footer__container container"
    >
      <a
        class="footer__logo logo"
        href="#"
      >
        <img
          alt="Логотип"
          class="logo__img"
          height="70"
          src="./img/svg/logo.svg"
          width="70"
        />
      </a>
      <div
        class="socials footer__socials"
      >
        <ul
          class="socials__list"
        >
          <li
            class="socials-item"
          >
            <a
              aria-label="facebook"
              class="socials__link"
              href="https://www.facebook.com/"
            >
              <svg
                aria-hidden="true"
                class="socials__icon"
                height="24"
                width="24"
              >
                <use
                  href="#icon-facebook"
                />
              </svg>
            </a>
          </li>
          <li
            class="socials-item"
          >
            <a
              aria-label="instagram"
              class="socials__link"
              href="https://www.instagram.com/"
            >
              <svg
                aria-hidden="true"
                class="socials__icon"
                height="24"
                width="24"
              >
                <use
                  href="#icon-instagram"
                />
              </svg>
            </a>
          </li>
          <li
            class="socials-item"
          >
            <a
              aria-label="twitter"
              class="socials__link"
              href="https://www.twitter.com/"
            >
              <svg
                aria-hidden="true"
                class="socials__icon"
                height="24"
                width="24"
              >
                <use
                  href="#icon-twitter"
                />
              </svg>
            </a>
          </li>
        </ul>
      </div>
      <section
        class="footer__nav-section footer__nav-section--info"
      >
        <h2
          class="footer__nav-title"
        >
          О нас
        </h2>
        <p
          class="footer__nav-content footer__nav-content--font-secondary"
        >
          Магазин гитар, музыкальных инструментов и гитарная мастерская 
          <br />
           в Санкт-Петербурге.
          <br />
          <br />
          Все инструменты проверены, отстроены 
          <br />
           и доведены до идеала!
        </p>
      </section>
      <section
        class="footer__nav-section footer__nav-section--links"
      >
        <h2
          class="footer__nav-title"
        >
          Информация
        </h2>
        <ul
          class="footer__nav-list"
        >
          <li
            class="footer__nav-list-item"
          >
            <a
              class="link"
              href="#top"
            >
              Где купить?
            </a>
          </li>
          <li
            class="footer__nav-list-item"
          >
            <a
              class="link"
              href="#top"
            >
              Блог
            </a>
          </li>
          <li
            class="footer__nav-list-item"
          >
            <a
              class="link"
              href="#top"
            >
              Вопрос - ответ
            </a>
          </li>
          <li
            class="footer__nav-list-item"
          >
            <a
              class="link"
              href="#top"
            >
              Возврат
            </a>
          </li>
          <li
            class="footer__nav-list-item"
          >
            <a
              class="link"
              href="#top"
            >
              Сервис-центры
            </a>
          </li>
        </ul>
      </section>
      <section
        class="footer__nav-section footer__nav-section--contacts"
      >
        <h2
          class="footer__nav-title"
        >
          Контакты
        </h2>
        <p
          class="footer__nav-content"
        >
          Санкт-Петербург
          ,
          <br />
          м. Невский проспект
          ,  
          <br />
          ул. Казанская 6
        </p>
        <div
          class="footer__nav-content"
        >
          <svg
            aria-hidden="true"
            class="footer__icon"
            height="8"
            width="8"
          >
            <use
              href="#icon-phone"
            />
          </svg>
          <a
            class="link"
            href="tel:8-812-500-50-50"
          >
            8-812-500-50-50
             
          </a>
        </div>
        <p
          class="footer__nav-content"
        >
          Режим работы:
          <br />
          <span
            class="footer__span"
          >
            <svg
              aria-hidden="true"
              class="footer__icon"
              height="13"
              width="13"
            >
              <use
                href="#icon-clock"
              />
            </svg>
            <span>
              с 11:00 до 20:00
               
            </span>
            <span>
              без выходных
            </span>
          </span>
        </p>
      </section>
    </div>
  </footer>
  
  
  
  
</div>
`;
